Week2/homework/kalb3/map-filter (2).js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+/* cSpell:disable */
+
+function doubleOddNumbers(numbers) {
+  numbers = numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```

Comment:
Nice use of chaining! Keep in mind that your code should also still be readable, if one line gets to long it sometimes is better to do the second function on a new line.